Sources/MapboxCoreNavigation/MapMatchingResult.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import MapboxNavigationNative
+import CoreLocation
+import Turf
+
+/// Provides information about the status of the enhanced location updates generated by the map matching engine of the Navigation SDK.
+public struct MapMatchingResult {
+    
+    /// The best possible location update, snapped to the route or map matched to the road if possible
+    public var enhancedLocation: LocationCoordinate2D
```

Comment:
What's wrong with `CLLocation`? I would like to have matched bearing and other fields. There is a way to convert `FixLocation` to `CLLocation`.